wasm.md

```diff
@@ -1531,6 +1496,238 @@ Start Function
          </moduleInst>
 ```
 
+Export
+------
+
+Exports make functions, tables, memories and globals available for importing into other modules.
```

Comment:
hmmm.. it also makes it available to be accessed by `Actions`. You decide whether to mention because `Action` belongs to `test.md`.

Reply:
I prefer to not mention actions here, like you say, because it belongs to `test.md`, i.e. the embedder.